sdk/node/testSrc/callbacks.js

```diff
@@ -11,7 +11,7 @@ const balanceByAssetAlias = (cb) => {
     let res = {}
 
     balances.items.forEach((item) => {
-      res[item.sum_by['asset_alias']] = item.amount
+      res[item.sumBy['assetAlias']] = item.amount
```

Comment:
I realize this is old code, but it's probably ok to do `item.sumBy.assetAlias` here.